lib/Twig/Environment.php

```diff
@@ -448,6 +450,8 @@ public function resolveTemplate($names)
      */
     public function clearTemplateCache()
     {
+        @trigger_error(sprintf('%s is deprecated and will be removed in Twig 2.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
I don't think we need to much details in Twig as all 1.x versions are basically interchangeable. Saying something was deprecated in 1.16 does not add anything useful.